docs/reference/components/sinks/sematext_logs.cue

```diff
@@ -0,0 +1,108 @@
+package metadata
+
+components: sinks: sematext_logs: {
+	title:             "Sematext Logs"
+	short_description: "Batches log events to [Sematext][urls.sematext] via the [Elasticsearch API][urls.sematext_es]."
+	long_description:  "[Sematext][urls.sematext] is a hosted monitoring platform based on Elasticsearch. Providing powerful monitoring and management solutions to monitor and observe your apps in real-time."
+
+	classes: {
+		commonly_used: false
+		egress_method: "batch"
+		function:      "transmit"
+		service_providers: ["Sematext"]
+	}
+
+	features: {
+		batch: {
+			enabled:      true
+			common:       false
+			max_bytes:    10490000
+			max_events:   null
+			timeout_secs: 1
+		}
+		buffer: enabled:      true
+		compression: enabled: false
+		encoding: codec: enabled: false
+		healthcheck: enabled: true
+		request: {
+			enabled:                    true
+			in_flight_limit:            5
+			rate_limit_duration_secs:   1
+			rate_limit_num:             5
+			retry_initial_backoff_secs: 1
+			retry_max_duration_secs:    10
+			timeout_secs:               60
+		}
+		tls: enabled: false
+	}
+
+	statuses: {
+		delivery:    "at_least_once"
+		development: "beta"
+	}
+
+	support: {
+		platforms: {
+			"aarch64-unknown-linux-gnu":  true
+			"aarch64-unknown-linux-musl": true
+			"x86_64-apple-darwin":        true
+			"x86_64-pc-windows-msv":      true
+			"x86_64-unknown-linux-gnu":   true
+			"x86_64-unknown-linux-musl":  true
+		}
+
+		requirements: []
+		warnings: []
+		notices: []
+	}
+
+	configuration: {
+		endpoint: {
+			common:      false
+			description: "The endpoint that will be used to send logs to. This option is required if `region` is not set."
```

Comment:
The code can handle `(None, None)` so I think neither of these option is required.

https://github.com/timberio/vector/blob/dbbed8db8c7b1fd6823180e1e37168bbc99148b1/src/sinks/sematext/logs.rs#L49

Reply:
Same for sematext_metrics